runtime/common/src/precompile/mod.rs

```diff
@@ -166,7 +170,8 @@ where
 				// HOMA,
 				EVM_ACCOUNTS,
 				/* HONZON
-				 * INCENTIVES */
+				 * INCENTIVES
+				 * XTOKENS */
```

Comment:
this will make it hard to test this on karura / acala testnet
one solution is enable it here, but have a migration to disable it with transaction-puase pallet
then we can unpause it on testnet
and for mainnet, we don't need a runtime upgrade to enable it